CONTRIBUTING.md

```diff
@@ -80,6 +88,9 @@ Other Tips
 
       $ git commit -m "Fixed typo [docs only]"
 
+- There is no naming standard for branches, but don't name your working 
```

Comment:
This is not needed here I feel, and it's already in the development workflow linked above.